src/Games/NexusMods.Games.FOMOD/CoreDelegates/UiDelegate.cs

```diff
@@ -133,7 +136,11 @@ public void UpdateState(FomodInstaller.Interface.ui.InstallerStep[] installSteps
             optionIdMappings
         );
 
-        var progress = Percent.CreateClamped(currentStepId, installSteps.Length);
+
+        // Progress is computed as currentIndex + 1 /(all steps before the current step + future visible steps + 1).
+        // The +1 are to avoid having the bar at 0% when we're at the first step, and to account for the extra finish screen.
```

Comment:
Why should the progress bar not be at 0% when the user is at the first step? The progress bar indicates how far along the user is with the installation process. 0% means the user hasn't done anything yet, since they are at the first step.

Reply:
When the bar is at 0% there is no information for the user regarding how much stuff is left since it makes no difference on whether it is 0/2 or 0/100. Setting it to 1/3 1/101 fixes this, giving the user an idea of how much each step counts towards the finish.

Reply:
The user will know how much each step counts towards the finish after the first step. Having the bar already filled to a certain point is more confusing than having it at 0%.

Reply:
The issue I'm specifically trying to solve is when the Fomod installer closes after the first step, with the user expecting to have much more than just the first step since the progress bar was at 0%.

Reply:
Ideally, the installer should jump to the "end" screen if that happens.

Reply:
Haven't managed to fix that yet. I can look into it more, but no guarantees.

Reply:
Had a chat with captainsandypants and he agreed with both starting with 1 for first page and counting the Finish page towards the total.